common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -62,6 +62,8 @@ public class CommonValue {
         public static final Integer ML_MODEL_GROUP_INDEX_SCHEMA_VERSION = 2;
         public static final Integer ML_MODEL_INDEX_SCHEMA_VERSION = 11;
         public static final String ML_CONNECTOR_INDEX = ".plugins-ml-connector";
+        public static final String ML_CONVERSATION_META_INDEX = ".plugins-ml-conversation-meta";
```

Comment:
Why do we need these two indices? I don't think we are using these?

Reply:
Removed these. I added them in a prior commit since they are still tracked in the security plugin: https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/tools/democonfig/SecuritySettingsConfigurer.java#L55-L56